test/pouch_cli_images_test.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"github.com/go-check/check"
+	"os/exec"
+	"strings"
+)
+
+// PouchImagesSuite is the test suite of help CLI.
+type PouchImagesSuite struct {
+}
+
+func init() {
+	check.Suite(&PouchImagesSuite{})
+}
+
+// SetUpTest does common setup in the beginning of each test.
+func (suite *PouchImagesSuite) SetUpTest(c *check.C) {
+	SkipIfFalse(c, IsLinux)
+}
+
+// TestPouchImages is to verify pouch images.
+func (suite *PouchImagesSuite) TestPouchImages(c *check.C) {
+	out, err := exec.Command("pouch", "images").Output()
+	c.Assert(err, check.IsNil)
+
+	if !strings.Contains(string(out), "IMAGE") {
```

Comment:
I have to say this test may not be so sufficient. I think checking `pouch images` after pulling an image is much better. WDYT? @zeppp

Reply:
Thanks for your advice!However,how can I know the time once image is pulled successfully?I can not check the err before pulling down.